pages/specification/release-notes/index.page.tsx

```diff
@@ -29,58 +39,77 @@ export async function getStaticProps() {
   };
 }
 
+interface CardProps {
+  title: string;
+  body: string;
+  headerSize: 'small';
+  bodyTextSize: 'small';
+  link: string;
+}
+
+const releaseNotes: CardProps[] = [
+  {
+    title: 'Draft 2020-12',
+    body: 'Draft 2020-12 release notes.',
+    headerSize: 'small',
+    bodyTextSize: 'small',
+    link: '/draft/2020-12/release-notes',
+  },
+  {
+    title: 'Draft 2019-09',
+    body: 'Draft 2019-09 release notes.',
+    headerSize: 'small',
+    bodyTextSize: 'small',
+    link: '/draft/2019-09/release-notes',
+  },
+  {
+    title: 'Draft 07',
+    body: 'Draft 07 release notes.',
+    headerSize: 'small',
+    bodyTextSize: 'small',
+    link: '/draft-07/json-schema-release-notes',
+  },
+  {
+    title: 'Draft 06',
+    body: 'Draft 06 release notes.',
+    headerSize: 'small',
+    bodyTextSize: 'small',
+    link: '/draft-06/json-schema-release-notes',
+  },
+  {
+    title: 'Draft 05',
+    body: 'Draft 05 release notes.',
+    headerSize: 'small',
+    bodyTextSize: 'small',
+    link: '/draft-05#explanation-for-lack-of-draft-05-meta-schema',
+  },
+];
+
 export default function ImplementationsPages({
   blocks,
   frontmatter,
-}: {
-  blocks: any;
-  frontmatter: any;
-}) {
+}: StaticProps) {
   const markdownFile = '_indexPage';
   return (
     <SectionContext.Provider value={null}>
       <Headline1>{frontmatter.title}</Headline1>
       <StyledMarkdown markdown={blocks.index} />
-      <StyledMarkdown markdown={blocks.body} />
+      {blocks.body && <StyledMarkdown markdown={blocks.body} />}
```

Comment:
I don't think you need to optionally render this, can you pls. explain why such change ?

Reply:
Actually, if you check the block prop in getStaticProps(), the `body:  bodyContent` is not currenty in use and is commented....

still I'm replacing it with the older change